bundle/EventDispatcher/SyliusEventDispatcher.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Netgen\Bundle\LayoutsSyliusBundle\EventDispatcher;
+
+use Sylius\Bundle\ResourceBundle\Controller\EventDispatcherInterface;
+use Sylius\Bundle\ResourceBundle\Controller\RequestConfiguration;
+use Sylius\Bundle\ResourceBundle\Event\ResourceControllerEvent;
+use Sylius\Component\Resource\Model\ResourceInterface;
+use Symfony\Contracts\EventDispatcher\EventDispatcherInterface as SymfonyEventDispatcherInterface;
+
+use function sprintf;
+
+final class SyliusEventDispatcher implements EventDispatcherInterface
+{
+    public function __construct(
+        private readonly EventDispatcherInterface $innerEventDispatcher,
```

Comment:
Do not use `private readonly` please, only `private`.

Reply:
Fixed in https://github.com/netgen-layouts/layouts-sylius/pull/11/commits/93ab79cc5e0f027f9222331aa0551e4358fa80db